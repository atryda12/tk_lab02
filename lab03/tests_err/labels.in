label : {int a = 3;}
label : {int a = 3;}