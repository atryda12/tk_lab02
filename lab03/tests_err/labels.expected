Error: Label 'label' already in use in this scope : line 2
Error: Variable 'a' already declared: line 2